### PR TITLE
Optional Job Attributes

### PR DESCRIPTION
**Pull request by akshaykrjain**

### CODE REVIEW

File: .github/workflows/docker.yaml [hunk 1]
```diff
@@ -3,10 +3,31 @@ name: Reusable CI workflow for Docker Build & Push to ECR
 on:
   workflow_call:
     inputs:
+      name: 
+        description: "Application Name overwrite (Default is the github repo name)"
+        required: false
+        type: string
+        default: ${{ github.event.repository.name  }}
       trivy_exit_code:
+        description: "if set to 0, workflow will ignore trivy scan errors.(not recommended)"
```
Comment (espizo) [inline .github/workflows/docker.yaml:12]: should be "if set to 0, workflow will ignore trivy scan errors (not recommended)"

File: .github/workflows/docker.yaml [hunk 2]
```diff
@@ -3,10 +3,31 @@ name: Reusable CI workflow for Docker Build & Push to ECR
 on:
   workflow_call:
     inputs:
+      name: 
+        description: "Application Name overwrite (Default is the github repo name)"
+        required: false
+        type: string
+        default: ${{ github.event.repository.name  }}
```
Comment (espizo) [inline .github/workflows/docker.yaml:10]: spacing

File: .github/workflows/docker.yaml [hunk 3]
```diff
@@ -3,10 +3,31 @@ name: Reusable CI workflow for Docker Build & Push to ECR
...
         required: false
         type: number
         default : 1
+      skip_docker:
+        description: "if set to true, workflow will skip docker build and ECR push"
+        required: false
+        type: boolean
+        default : false
+      skip_irsa:
+        description: "if set to true, workflow will skip IRSA"
```
Comment (espizo) [inline .github/workflows/docker.yaml:22]: should be "if set to true, workflow will skip IRSA creation"

File: .github/workflows/docker.yaml [hunk 4]
```diff
@@ -28,17 +49,17 @@ jobs:
         id: create-ecr-repo
         uses: int128/create-ecr-repository-action@v1
         with:
-          repository: ${{ github.event.repository.name  }}  
+          repository: ${{ inputs.name  }}
```
Comment (espizo) [inline .github/workflows/docker.yaml:52]: spacing

**Review by espizo — DISMISSED** [review]

File: .github/workflows/docker.yaml (hunk 2 above)
Comment (akshaykrjain) [reply .github/workflows/docker.yaml]: fixed

File: .github/workflows/docker.yaml (hunk 4 above)
Comment (akshaykrjain) [reply .github/workflows/docker.yaml]: fixed

**Review by espizo — APPROVED** [review]

**Pull request merged**